ios/fastlane/Fastfile

```diff
@@ -17,26 +17,37 @@ default_platform(:ios)
 
 skip_docs
 
-# CONFIG VARIABLES
-app_name      = 'ReactNativeBase'                 # The canonical project name
-xcodeproj     = app_name+'.xcodeproj'             # The xcodeproj name eg: ReactNativeBase.xcodeproj
-workspace     = app_name+'.xcworkspace'           # The xworkspace name here eg: ReactNativeBase.xcworkspace
-team_id       = ENV["APPLE_TEAM_ID"]              # The organization's team id in the Apple Developer portal
-cert          = ENV["APPLE_CERT"]                 # Local path to distribution certificate file to be used for signing the build 
-key           = ENV["APPLE_KEY"]                  # Private key (.p12 file) used for encrypting certificate
-key_pwd       = ENV["APPLE_KEY_PASSWORD"]         # Password to private key file
-s3_region     = ENV["AWS_REGION"]                 # AWS authentication params
-s3_key        = ENV["AWS_ACCESS_KEY_ID"]          #
-s3_secret_key = ENV["AWS_SECRET_ACCESS_KEY"]      #
-s3_bucket     = ENV["BUILD_BUCKET"]               # AWS S3 bucket and folder to upload builds to (for Adhoc builds)
-folder        = ENV["FOLDER"]                     # 
-slack_url     = ENV["SLACK_URL"]                  # Slack webhook url and channel name for sending notifications upon completion
-slack_channel = ENV["SLACK_CHANNEL"]              #
+# REQUIRED ENVIRONMENT VARIABLES
+# ==============================
+# FASTLANE_USER                                 : Your App Store Connect / Apple Developer Portal id used for managing certificates and submitting to the App Store
+# FASTLANE_PASSWORD                             : Your App Store Connect / Apple Developer Portal password, usually only needed if you also set the FASTLANE_USER variable
```

Comment:
This is not being used anymore, probably should be replaced with APP_STORE_CONNECT_API_KEY_FILE